lib/ESP8266WebServer/src/ESP8266WebServer.h

```diff
@@ -129,7 +129,7 @@ template<typename T> size_t streamFile(T &file, const String& contentType){
     sendHeader("Content-Encoding", "gzip");
   }
   send(200, contentType, "");
-  return _currentClient.write(file, HTTP_DOWNLOAD_UNIT_SIZE);
+  return _currentClient.write(file);
```

Comment:
Why does this need to change?

Reply:
write no longer accepts the size as a parameter, it comes from the file stream directly (that version of write is depreciated)